AlexDanoff/src/math_function.py

```diff
@@ -27,46 +26,87 @@ def change_original(o_file, changed_math_string):
     return edited
 
 
-def index_spacing(file, out_file):
-    used = open(file).read()
+def formatting(file_str):
+    # Proper spacing for the indexes and gets rid of all non begin{eq end{eq text
+
+    commands = r'\\[a-zA-Z]+'
 
     updated = []
-    IND_START = r'\index{'
+    IND_START = r'\index\{'
```

Comment:
{ is a special character in regex. However, this time the code does the same thing.